cmake/CPackGeneratorConfig.cmake

```diff
@@ -5,5 +5,5 @@ message(STATUS "Configuring for generator ${CPACK_GENERATOR} and TOPLEVEL_TAG ${
 
 if("${CPACK_TOPLEVEL_TAG}" STREQUAL "Linux-Source")
   message(STATUS "Adding python dependency")
-  set(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS},python3")
+  set(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS},python")
```

Comment:
Can you explain this one? Is this installing both `python3` and `python2`?

Reply:
The `python` dependency is for python 2.7. The `python3` dependency is for python 3.8. We actually need both to pass lintian, as we have some python scripts which start with `#!/usr/bin/env python` and with `#!/usr/bin/env python3`.